app/concerns/exportable_as_csv.rb

```diff
@@ -17,9 +17,16 @@ def self.to_csv
         csv << exportable_attributes.values
 
         all.each do |item|
-          csv << exportable_attributes.keys.map { |attr| item.send(attr) }
+          csv << exportable_attributes.keys.map do |attr|
+            safe_value = prevent_csv_code_injection(item.send(attr))
+            safe_value
```

Comment:
Doesn't look like you need the intermediary variable `safe_value` here.

Reply:
also true :)